src/lib.rs

```diff
@@ -23,6 +23,8 @@ pub mod tzkey;
 pub mod urdna2015;
 pub mod vc;
 
+pub static SSI_INFO: &str = concat!(env!("CARGO_PKG_VERSION"), "/", env!("CARGO_PKG_NAME"));
```

Comment:
I would rename that variable to just `USER_AGENT` or something like that, because it's not really information about ssi but about what binary is being built. But that's just nitpicking

Reply:
+1 to rename.

When I build didkit-cli with ssi at this branch, and resolve a `did:web`, I see in the server request log user-agent "0.2.2/ssi". So it looks like these variables are about the package this code is in rather than the binary being built, unless I'm missing something. There is also a `CARGO_CRATE_NAME` (= "ssi" here) and `CARGO_BIN_NAME` (undefined here).  Also the tokens should be ordered the other way (e.g. "ssi/0.2.2")

https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-crates